DBMS/JDBC/ConnectionUtil.java

```diff
@@ -0,0 +1,32 @@
+package jdbcassignment;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionUtil {
+
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final String USER = "root";
+    private static final String PASSWORD = "mysql";
+
+    private Connection con = null;
+
+    public Connection getConnection() {
+	/* register driver */
+	try {
+	    Class.forName("com.mysql.jdbc.Driver");
+	} catch (ClassNotFoundException e) {
+	    e.printStackTrace();
+	}
+
+	/* open connection */
+	try {
```

Comment:
Use single try block and multiple catch block for Class.forName and below statement.